src/main/kotlin/game/racingcar/domain/Car.kt

```diff
@@ -0,0 +1,18 @@
+package game.racingcar.domain
+
+import game.racingcar.strategy.MoveStrategy
+
+class Car {
+    var blackBox = arrayListOf<Int>()
+    var position: Int = 0
+
+    fun move(moveStrategy: MoveStrategy): Int {
+        position = moveStrategy.move(position)
+        blackBox.add(position)
+        return position
+    }
+
+    fun history(time: Int): Int {
+        return blackBox[time]
+    }
```

Comment:
```suggestion
    fun history(time: Int): Int = blackBox[time]
```

해당 경우에는 문(statement) 보다 간결성을 위하여 식(expression)으로 변경하는 게 좋지 않을까요? 🤔